r/tests/testthat/latin1.R

```diff
@@ -15,7 +15,7 @@
 # specific language governing permissions and limitations
 # under the License.
 
-x <- iconv("Veitingasta�ir", to = "latin1")
+x <- iconv("Veitingasta�ir", to = "latin1")
```

Comment:
Was this change intentional?

Reply:
Whoa, no

Reply:
Yeah you need to open this in a text editor that won't reencode to UTF-8 (`vim` may be suited for this), or otherwise set the encoding in the editor to latin1 (or whatever its official name is).

Reply:
Fixed in e894bdd